File: Upcoder/Khối 11/Chương 3 Cấu trúc lặp/Tính tổng cơ bản/K11_TinhTong.py
n = int(input())  # Nhập giá trị n từ bàn phím
s = 0  # Khởi tạo tổng bằng 0

if n < 0:
    print("NHAP SAI")  # Nếu n nhỏ hơn 0 thì xuất "NHAP SAI"
else:
    for i in range(n + 1):  # Vòng lặp từ 0 đến n
        s += i  # Cộng dồn các giá trị từ 0 đến n vào s
    print(s)  # In tổng giá trị

File: Upcoder/Khối 11/Chương 3 Cấu trúc lặp/Tính tổng cơ bản/K11_TinhTong.cs
using System;

namespace TongSo
{
    public class Program
    {
        public static void Main(string[] args)
        {
            // Nhập giá trị n từ bàn phím
            int n = int.Parse(Console.ReadLine());

            int s = 0; // Khởi tạo tổng bằng 0

            // Kiểm tra nếu n < 0
            if (n < 0)
            {
                Console.WriteLine("NHAP SAI");
            }
            else
            {
                // Vòng lặp từ 0 đến n
                for (int i = 0; i <= n; i++)
                {
                    s += i; // Cộng dồn các giá trị từ 0 đến n vào s
                }

                // In ra tổng giá trị
                Console.WriteLine(s);
            }
        }
    }
}
